GoogleForCreators/web-stories-wp
# Updated every reference from cover to poster where applicable

## Context

To avoid confusion, we're renaming "Cover image" to "Poster image" - and changing every reference to "story cover image" to simply be "poster image".

Also fixing a few typos and other stuff while at it (mostly in tests).

## User-facing changes

Input text has changed next to the image selector as well as messaging in pre-publish panel:

| | Before | After |
|-|-|-|
| Input | ![Screen Shot 2021-02-12 at 14 35 38](https://user-images.githubusercontent.com/637548/107814377-ce832300-6d3f-11eb-82c3-8f8c7f28a177.png) | ![Screen Shot 2021-02-12 at 15 12 00](https://user-images.githubusercontent.com/637548/107817865-b5c93c00-6d44-11eb-8912-8ad8edb4af5a.png) |
| Missing image | ![Screen Shot 2021-02-12 at 14 35 43](https://user-images.githubusercontent.com/637548/107814376-ce832300-6d3f-11eb-957c-0e074678f755.png) | ![Screen Shot 2021-02-12 at 14 35 15](https://user-images.githubusercontent.com/637548/107814378-cf1bb980-6d3f-11eb-8c51-cac4f62ecfe7.png) |
| Wrong size | ![Screen Shot 2021-02-12 at 14 35 56](https://user-images.githubusercontent.com/637548/107814375-ce832300-6d3f-11eb-9325-b2938dbde710.png) | ![Screen Shot 2021-02-12 at 14 36 29](https://user-images.githubusercontent.com/637548/107814374-cdea8c80-6d3f-11eb-84f2-e7e51264d4d0.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Create new story
2. Go to document panel
3. Observe label next to poster image input
4. Go to prepublish panel
5. Observe message around missing poster image

### UAT




## Checklist




---

Fixes #6370
